src/components/Main.tsx

```diff
@@ -2,34 +2,37 @@ import { Link, Outlet } from "react-router-dom";
 import { useAppServices } from "./AppServicesContext";
 import logo from "./logo.svg";
 import "./Main.css";
+import { SingletonEditorProvider } from "./SingletonEditor";
 
 export function Main() {
   const {
     appConfiguration: { loginPageUrl },
   } = useAppServices();
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <div className="flex-row-center h-full">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1>Editors WebApp</h1>
+    <div className="App dp-library dp-wrapper">
+      <header>header</header>
+      <main>
+        <SingletonEditorProvider>
+          <Outlet />
+        </SingletonEditorProvider>
+      </main>
+      <footer>
+        <div className="ed-cta-footer">
+          <button
+            type="button"
+            className="dp-button button-medium secondary-green"
+          >
+            Salir y editar luego
+          </button>
```

Comment:
It will not work because the footer is out of the SingletonEditorProvider, so it does not have access to the singleton editor hooks.

Reply:
If the footer buttons are related to the editor (save, continue, etc). 

In my opinion, you have to replicate the same approach that we take with the `EditorTopBar.tsx`, I mean, it should be an _editor's bottom bar_ in place of the application footer.

Reply:
:+1:  Ok